customer-bill/src/main/java/org/fiware/tmforum/customer_bill/TMForumMapper.java

```diff
@@ -0,0 +1,124 @@
+package org.fiware.tmforum.customer_bill;
+
+import org.fiware.customer_bill.model.*;
+import org.fiware.tmforum.customer_bill.domain.*;
+import org.fiware.tmforum.customer_bill.domain.customer_bill.AppliedCustomerBillingRate;
+import org.fiware.tmforum.customer_bill.domain.customer_bill.CustomerBill;
+import org.fiware.tmforum.customer_bill.domain.customer_bill.CustomerBillOnDemand;
+import org.fiware.tmforum.mapping.MappingException;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+
+/**
+ * Mapper between the internal model and api-domain objects
+ */
+@Mapper(componentModel = "jsr330")
+public interface TMForumMapper {
+
+    String ID_TEMPLATE = "urn:ngsi-ld:%s:%s";
+
+    CustomerBillVO map(CustomerBill customerBill);
+    CustomerBill map(CustomerBillVO customerBillVO);
+
```

Comment:
In order to allow referencing between objects, the top level accessible objects(where we generate the ID anyways) should get ngsi-compliant ids. Please check Individual and Organization in the party-module